metadata-io/src/main/java/com/linkedin/metadata/search/transformer/SearchDocumentTransformer.java

```diff
@@ -119,6 +119,40 @@ public void setValue(final SearchableFieldSpec fieldSpec, final List<Object> fie
     }
   }
 
+  public void setSearchScoreValue(final SearchScoreFieldSpec fieldSpec, final List<Object> fieldValues,
+      final ObjectNode searchDocument, final Boolean forDelete) {
```

Comment:
What does "forDelete" mean?

Reply:
This is something gabe added before. It is to support deleting aspects, so it sends over empty jsons for the fields that are being deleted so it gets deleted on elasticsearch side.

Reply:
ah... okay